test/unit/custom_spec.js

```diff
@@ -107,3 +107,104 @@ describe("custom canvas rendering", function () {
       .catch(done.fail);
   });
 });
+
+describe("alternate document context", function () {
+  const FontFace = global.FontFace;
+
+  let CanvasFactory;
+
+  beforeAll(() => {
+    global.FontFace = function MockFontFace(name) {
+      this.family = name;
+    };
+    CanvasFactory = isNodeJS ? new NodeCanvasFactory() : new DOMCanvasFactory();
+  });
+
+  afterAll(() => {
+    global.FontFace = FontFace;
+    CanvasFactory = null;
+  });
+
+  it("should use given document for loading fonts", async function () {
```

Comment:
Probably clearer to call this test `should use given document for loading fonts (with Font Loading API)`.

Reply:
I'll update the test names to use those labels.